diff --git a/node_modules/metro/src/ModuleGraph/node-haste/node-haste.js b/node_modules/metro/src/ModuleGraph/node-haste/node-haste.js
index 631014d..fb68add 100644
--- a/node_modules/metro/src/ModuleGraph/node-haste/node-haste.js
+++ b/node_modules/metro/src/ModuleGraph/node-haste/node-haste.js
@@ -153,7 +153,7 @@ exports.createResolveFn = function (options) {
       rootDir: "",
     }),
     nodeModulesPaths: options.nodeModulesPaths,
-    preferNativePlatform: true,
+    preferNativePlatform: false,
     projectRoot: "",
     resolveAsset: (dirPath, assetName, extension) => {
       const basePath = dirPath + path.sep + assetName;
diff --git a/node_modules/metro/src/node-haste/DependencyGraph.js b/node_modules/metro/src/node-haste/DependencyGraph.js
index adc90d6..6211366 100644
--- a/node_modules/metro/src/node-haste/DependencyGraph.js
+++ b/node_modules/metro/src/node-haste/DependencyGraph.js
@@ -146,7 +146,7 @@ class DependencyGraph extends EventEmitter {
       moduleCache: this._moduleCache,
       moduleMap: this._moduleMap,
       nodeModulesPaths: this._config.resolver.nodeModulesPaths,
-      preferNativePlatform: true,
+      preferNativePlatform: false,
       projectRoot: this._config.projectRoot,
       resolveAsset: (dirPath, assetName, extension) => {
         const basePath = dirPath + path.sep + assetName;
